Apps/System/systemapp.c

```diff
@@ -23,6 +23,10 @@ static Window *s_about_window;
 static Layer *s_aboutCanvas_layer;
 static ScrollLayer *s_about_scroll;
 static void about_update_proc(Layer *layer, GContext *nGContext);
+static GBitmap *logo_color;
+static GBitmap *rocket_color;
+static GBitmap *logo_bw;
+static GBitmap *rocket_color;
```

Comment:
double definition of rocket_color?

Reply:
Yikes, didn't mean for that to get through! Fixed.